app/views/match_mailer/match_confirmation_instructions.mjml

```diff
@@ -16,13 +17,13 @@
     <mj-text padding-bottom="0px">
       <strong>Vous ne souhaitez plus être informé de doses disponibles ?</strong>
     </mj-text>
-    <mj-button href="<%= edit_matches_users_url(match_confirmation_token: @match_confirmation_token) %>" padding-bottom="0px">
+    <mj-button href="<%= confirm_destroy_profile_url(authentication_token: authentication_token) %>" padding-bottom="0px">
       Supprimer mon compte
     </mj-button>
     <mj-text>
       <strong>Si le lien ne fonctionne pas</strong>, copiez et collez l’adresse suivante dans votre navigateur :
       <br />
-      <%= edit_matches_users_url(match_confirmation_token: @match_confirmation_token) %>
```

Comment:
I replaced this in order to remove the nested controllers in `Match` and `SlotAlert` controllers.